<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actDetailWindow</class>
 <widget class="QDialog" name="actDetailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>活动详情</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="tagTitle">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>标签</string>
   </property>
  </widget>
  <widget class="QLabel" name="detailTitle">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>详情</string>
   </property>
  </widget>
  <widget class="QLabel" name="peopleTitle">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参与人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signBntm">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>610</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>报名活动</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="detailText">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>291</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="tagIndetail">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="uniTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_frame">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>80</y>
     <width>451</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>501</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>1</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>label_frame</zorder>
  <zorder>tagTitle</zorder>
  <zorder>detailTitle</zorder>
  <zorder>peopleTitle</zorder>
  <zorder>signBntm</zorder>
  <zorder>listWidget</zorder>
  <zorder>detailText</zorder>
  <zorder>tagIndetail</zorder>
  <zorder>uniTitle</zorder>
  <zorder>label_pic</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
